upload-s3/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Upload S3'
+description: 'Uploads package to S3'
+inputs:
+  bucket:
+    required: true
+  product:
+    required: true
+  branch:
+    required: true
+  build:
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Create and release

```

Comment:
`Nit`
This step name might be clearer if it just says `Upload release` - ie. we aren't actually creating anything just uploading a file.